src/hooks/useDrillDownUrl.ts

```diff
@@ -0,0 +1,28 @@
+import {useSelector} from 'react-redux'
+import { Store } from 'interfaces/store'
```

Comment:
Should use relative import (https://github.com/microsoft/TypeScript/issues/15479)